.gitignore

```diff
@@ -7,8 +7,10 @@
 Files.App.OpenDialog*.dll
 Files.App.SaveDialog*.dll
 Files.App.Launcher.exe
+Files.Extensions/nul
```

Comment:
```suggestion
```

Reply:
What is this?

Reply:
I have no idea.

Reply:
It's being excluded right now. https://github.com/files-community/Files/blob/a8942f0341da13b0760de37fcdfefe6828fefe3b/.gitignore#L334
Two years ago I would've been able to comprehend the Files codebase, but I'm starting to lose that ability... 🫠

Reply:
> Files.Extensions

This namespace didnt existed when i started contributing. I remeber there was but it must be a few years ago.

Reply:
Yeah me too